traffic_monitor/static/style.css

```diff
@@ -44,11 +44,11 @@ tbody tr:nth-child(odd) {
 	background: #fff;
 }
 .error {
-	background-color: #f00;
+	background-color: #f00!important;
 }
 
 .warning {
-	background-color: #f80;
```

Comment:
Is this fixing a bug?

Reply:
Yes. The selector is less specific than `tbody tr:nth-child(even)`, so it's getting overridden.